License

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
Please replace the [year] and [fullname] placeholders with the actual year and your full name in order to make the license yours and valid.